app/assets/stylesheets/style.css

```diff
@@ -75,6 +75,29 @@ h6.dropdown-header {
   border: none;
 }
 
+
+h1 {
+  font-size: 1.95rem;
+}
+
+@media only screen and (max-width: 400px) {
+    h1 {
+      font-size: 1.6rem;
+    }
+    h2 {
+      font-size: 1.4rem;
+    }
+    h3 {
+      font-size: 1.3rem;
+    }
+    h4 {
+      font-size: 1.25rem;
+    }
+    h5 {
+      font-size: 1.2rem;
+    }
+}
+
```

Comment:
This looks really great @illyShelly. Is it possible to move these styles to `note.css`? We don't know what these styles would look like on other pages so it's better to only apply them to the notes, questions and wiki pages for now. Thanks!

Reply:
All you have to do is target #note-show and #main-content `heading` tags.

Reply:
HI @TildaDares , thank you for your time, really. I understand. Media query would affect everything. (haven't push yet)

I removed tag `h1` to `note.css` as below - which works (.note-show class and #main-content as id). What I cannot do and doesn't work **in plain css file** are media query. If you know how to do it, please let me know 🙂 

_As I read whether is possible to apply media query for particular class or id_: "Media queries should always be an outer shell, encasing the inside to be applied under that condition. In SASS and LESS, however, it's fully valid."

This is what I wanted to do - to avoid to apply media query for all pages but just for **note-show and media-content**: 

```
@media only screen and (max-width: 400px) {
   .note-show, #media-content {
      h1 {
      font-size: 1.6rem;
      }
      h2 {
        font-size: 1.4rem;
      }
      h3 {
        font-size: 1.3rem;
      }
      h4 {
        font-size: 1.25rem;
      }
      h5 {
        font-size: 1.2rem;
      }
}
```

This works and probably I make this as the change in the end:
```
.note-show h1, #main-content h1 {
  font-size: 1.95rem;
}
```

Reply:
@illyShelly The second option is what I had in mind. Also, you don’t have to add it to the note.css file anymore. You can leave it in style.css. Thanks!!

Reply:
Ok, I pushed :) ...I found out solution to the problem above, but it will make non-significant change in the app.

Reply:
Hi @illyShelly, you haven't added this code.
```
@media (max-width:400px) {
  .note-show h1, #main-content h1 {
    font-size: 1.6rem;
  }
  .note-show h2, #main-content h2 {
    font-size: 1.4rem;
  }
  .note-show h3, #main-content h3 {
    font-size: 1.3rem;
  }
  .note-show h4, #main-content h4 {
    font-size: 1.25rem;
  }
  .note-show h5, #main-content h5 {
    font-size: 1.2rem;
  }
}
```